int8                idx
geometry_msgs/Point position
